lib/matplotlib/textpath.py

```diff
@@ -382,7 +383,10 @@ def get_glyphs_tex(self, prop, s, glyph_map=None,
                     charcode = glyph
 
                 if charcode is not None:
-                    glyph0 = font.load_char(charcode, flags=ft2font_flag)
+                    if use_glyph:
```

Comment:
The `charcode` is set right before we reach this condition `if charcode is not None:`
Therefore, it seems like the right spot to decide which font method to use to load the `charcode`.